controller/cache/cluster.go

```diff
@@ -349,6 +350,12 @@ func (c *clusterInfo) sync() (err error) {
 		return err
 	}
 	c.serverVersion = version
+	groups, err := c.kubectl.GetAPIGroups(config)
```

Comment:
I think `kubectl.GetAPIGroups` is redundant to the call right below: `apis, err := c.kubectl.GetAPIResources`, so we can avoid this altogether and use `apis`

Reply:
I had to introduce `kubectl.GetAPIGroups`  because `kubectl.GetAPIResources` and `apis` contains only preferred versions of each group. We need all group versions e.g. `networking.k8s.io/v1beta1`, `networking.k8s.io/v1beta2` etc, so I had to load that in a separate call.